osu.Game.Rulesets.Osu/Mods/OsuModFlashlight.cs

```diff
@@ -21,42 +20,56 @@ public class OsuModFlashlight : ModFlashlight<OsuHitObject>, IApplicableToDrawab
     {
         public override double ScoreMultiplier => 1.12;
 
-        private const float default_flashlight_size = 180;
-
         private const double default_follow_delay = 120;
 
+        [SettingSource("Follow delay", "Milliseconds until the flashlight reaches the cursor")]
+        public BindableNumber<double> FollowDelay { get; } = new BindableDouble(default_follow_delay)
+        {
+            MinValue = default_follow_delay,
+            MaxValue = default_follow_delay * 10,
+            Precision = default_follow_delay,
+        };
+
+        [SettingSource("Change radius based on combo", "Decrease the flashlight radius as combo increases.")]
+        public override BindableBool ChangeRadius { get; } = new BindableBool
+        {
+            Default = true,
+            Value = true
+        };
+
+        [SettingSource("Initial radius", "Initial radius of the flashlight area.")]
+        public override BindableNumber<float> InitialRadius { get; } = new BindableNumber<float>
+        {
+            MinValue = 90f,
+            MaxValue = 360f,
+            Default = 180f,
+            Value = 180f,
+            Precision = 5f
```

Comment:
[Still not sure about these being raw sizes rather than multipliers to the default 180.](https://github.com/ppy/osu/pull/16473#discussion_r785444793) This isn't done in any other mod yet.

Would you have anything against rescaling this setting for all rulesets to be a "multiplier" applied to the default, so in this case the range would be 0.5x to 2x, with the default 1x internally meaning 180?

Reply:
if you give me an example to work with then no i dont mind doing that

Reply:
Not sure what "example" is required to apply this change. Divide these values by 180 and pass the multiplier to `ModFlashlight` to multiply the constant 180 radius with?

Reply:
> Not sure what "example" is required to apply this change. Divide these values by 180 and pass the multiplier to `ModFlashlight` to multiply the constant 180 radius with?

yeah but its a different amount for each mode  so im not sure how id do it

Reply:
Define a protected abstract property that each mod would override to put the normal flashlight radius?

Reply:
ok yeah i think i get it now

Reply:
@bdach I can i mark this resolved now?

Reply:
Marking conversation as resolved will be done by the reviewer at the point of visting this PR again.